libs/community/langchain_community/llms/azureml_endpoint.py

```diff
@@ -53,12 +53,12 @@ def call(
 
 
 class AzureMLEndpointApiType(str, Enum):
-    """Azure ML endpoints API types. Use `realtime` for models deployed in hosted
+    """Azure ML endpoints API types. Use `dedicated` for models deployed in hosted
     infrastructure, or `serverless` for models deployed as a service with a
     pay-as-you-go billing or PTU.
     """
 
-    realtime = "realtime"
+    dedicated = "dedicated"
```

Comment:
this is a breaking change. can we keep realtime and dedicated and treat them the same (and update all the docs to use dedicated)?